golden/golden.go

```diff
@@ -22,6 +22,16 @@ type helperT interface {
 	Helper()
 }
 
+// NormalizeCRLFToLF enables end-of-line normalization for actual values passed
+// to Assert and String, as well as the values saved to golden files with
+// -test.update-golden.
+//
+// Defaults to true. If you use the core.autocrlf=true git setting on windows
+// you will need to set this to false.
+//
+// The default value may change in a future major release.
+var NormalizeCRLFToLF = true
```

Comment:
What about adding another flag like `test.update-golden` ? You can then control the CRLF from command line (or Makefile) depending you are on windows or not?

Reply:
Thanks that makes sense 👍